src/app-logic/l10n.js

```diff
@@ -10,28 +10,38 @@ import {
   PSEUDO_STRATEGIES_DIRECTION,
 } from 'firefox-profiler/utils/l10n-pseudo';
 
-// This contains the locales we support in the production. Don't forget to update
-// the array RTL_LOCALES when adding a RTL locale, if necessary.
-// AVAILABLE_STAGING_LOCALES is replaced, using webpack's DefinePlugin, by all
-// available locales, when running with L10N=1, using the "l10n" versions of the
-// yarn scripts. Especially this is done when building and deploying the l10n
-// branch in netlify.
-export const AVAILABLE_LOCALES: Array<string> = AVAILABLE_STAGING_LOCALES || [
-  'de',
-  'el',
-  'en-GB',
-  'en-US',
-  'es-CL',
-  'fr',
-  'ia',
-  'it',
-  'nl',
-  'pt-BR',
-  'sv-SE',
-  'uk',
-  'zh-CN',
-  'zh-TW',
-];
+// This object contains the locales that are complete at at least 90% and
+// that enable by default. Don't forget to update the array RTL_LOCALES when
+// adding a RTL locale, if necessary.
+// The localized names are copied from the select box at the bottom of
+// https://addons.mozilla.org/. We use the language names in their own language.
```

Comment:
Let's not use that list, it's not great…

This is the one I created for Firefox prefs
https://mozilla-l10n.github.io/firefox-languages/

Reply:
ah, interesting... The one on AMO seemed better than the one on mozilla.org.

May I ask what's better in that one? For instance I would say that `Português (do Brasil)` is better than `Português (BR)`

Reply:
We have way too many sources for language names :-( 

mozilla.org gets it from [product-details](https://github.com/mozilla-releng/product-details/blob/production/public/1.0/languages.json), AMO maintains [its own](https://github.com/mozilla/addons-frontend/blob/master/src/amo/languages.js).

This one is manually curated, and was built analyzing the list of languages we have in Firefox (yet another source…) and CLDR. And, as usual when it comes to languages, there are a lot of pitfalls (e.g. [Chinese](https://bugzilla.mozilla.org/show_bug.cgi?id=1749367)), with names that are politically charged.